cdap-docs/reference-manual/source/release-notes.rst

```diff
@@ -30,6 +30,491 @@ Cask Data Application Platform Release Notes
+  and Hive table when creating a partitioned file set.
+
+- :cask-issue:`CDAP-7597` - Added configuring the CDAP Explore database and table name for
+  a dataset using dataset properties.
+
+- :cask-issue:`CDAP-7676` - Added a prototype implementation for a rolling HDFS log
+  appender.
+
+- :cask-issue:`CDAP-7683` - Added a tool that pre-builds and loads the HBase coprocessors
+  required by CDAP onto HDFS.
+
+- :cask-issue:`CDAP-7685` - Provided SPI hooks that users can implement for performing
+  HBase DDL operations.
+
+- :cask-issue:`CDAP-8021` - Added the management of preferences at the application and
+  program levels.
+
+- :cask-issue:`CDAP-8025` - Added a tool to check a cluster's replication status.
+
+- :cask-issue:`CDAP-8032` - CDAP context methods will now be retried according to a
+  program's retry policy.
+
+- :cask-issue:`CDAP-8036` - Added "fat" JARs containing all of the coprocessor classes and
+  dependencies to the cdap-hbase-compat packages.
+
+- :cask-issue:`CDAP-8037` - Added a ``master.manage.hbase.coprocessors`` setting that can be
+  set to false on clusters where the CDAP coprocessors are deployed on every HBase node.
+
+- :cask-issue:`CDAP-8061` - Moved the Cask Market Path to the ``cdap-defaults.xml`` file.
+
+- :cask-issue:`CDAP-8070` - Added control of group ownership and permissions through
+  dataset properties.
+
+- :cask-issue:`CDAP-8075` - The CDAP UI added one-step deploy wizards for the Cask Market.
+
+- :cask-issue:`CDAP-8108` - Refactored the CDAP Log Appender to use Logback's Appender
+  interface.
+
+- :cask-issue:`CDAP-8110` - Added support for fine-grained impersonation at the CDAP
+  application, dataset, and stream level.
+
+- :cask-issue:`CDAP-8122` - Made headers and the request/response bodies available in
+  audit logs for certain RESTful endpoints.
+
+- :cask-issue:`CDAP-8198` - The CDAP UI added a dataset detail view.
+
+- :cask-issue:`CDAP-8199` - The CDAP UI added a stream detail view.
+
+- :cask-issue:`CDAP-8203` - The CDAP UI added a "call-to-action" modal dialog after entity
+  creation.
+
+- :cask-issue:`CDAP-8214` - The CDAP UI added a dataset overview.
+
+- :cask-issue:`CDAP-8217` - The CDAP UI added a stream overview.
+
+- :cask-issue:`CDAP-8231` - The log file cleaner thread will remove metadata and, for
+  successfully deleted metadata entries, it will delete the corresponding log files. The log
+  file cleaner thread will only remove the metadata entries for the old (pre-4.1.0) log
+  format.
+
+- :cask-issue:`CDAP-8261` - Logs collected by the CDAP Log Appender will be stored at a
+  common ``<cdap>/logs`` path, owned by the cdap user. For security, it is readable only by
+  the cdap user.
+
+- :cask-issue:`CDAP-8282` - CDAP UI now has a View Events "fast-action" dialog for
+  streams.
+
+- :cask-issue:`CDAP-8292` - Added support for CDH 5.10.0.
+
+- :cask-issue:`CDAP-8355` - Impersonated namespace can be configured to disallow
```

Comment:
This is hard to follow. Perhaps:
"Added a parameter <parameter-name> in namespace config which determines whether explore queries can run as namespace user", but should explain even more clearly